Docker/local_run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+/opt/sd/launch --local-mode --local-build-json "$1" --local-job-name "$2" --api-uri "$3" --emitter /sd/emitter 0 &
```

Comment:
What is this `0` argument?

Reply:
Because launcher doesn't use `buildID` in local-mode, pass `0` to launcher as dummy.

Reply:
I think we should add a comment for this.

Reply:
Added comment